packages/@react-spectrum/s2/src/ClearButton.tsx

```diff
@@ -34,31 +34,37 @@ interface ClearButtonStyleProps {
 interface ClearButtonRenderProps extends ButtonRenderProps, ClearButtonStyleProps {}
 interface ClearButtonProps extends ButtonProps, ClearButtonStyleProps {}
 
+const visibleClearButton = style<ClearButtonRenderProps>({
+  ...focusRing(),
```

Comment:
The focus ring color isn't visible when the tag is selected + emphasized, forget what we've done in cases like thing (feels like thats partially why halo focus rings are a thing?)